tapering_window.py

```diff
@@ -72,12 +70,3 @@ def tapering_window(tseries,method="Hanning"):
         tseries_tapered=tseries*wind_kaiser
         print("kaiser window is applied.....")
     return tseries_tapered
-
-Data_Path="/home/jithin/Desktop/ADCP_Processing_Tool/adps-main/tests/BGS11000.000"
```

Comment:
Hey, 

It is throwing a error below..

NameError: name 'press' is not defined

I would suggest commenting the below part from this code. You can  write as a separate code to test the tapering function.

Reply:
I have removed the testing part from this code. I think the pull request is not yet merged. @shehulthinvrex please check.

Reply:
Ok, the revised code is working for me.

Reply:
Subeesh, if the code is working in your pc. You can resolve the conversation.

Reply:
It is not yet merged i think. I am getting same error after pulling. 
"tapered_series=tapering_window(press)" This part is still there in the main folder.